compiler-core/src/language_server/tests/hover.rs

```diff
@@ -65,6 +76,269 @@ fn(Int) -> Int
     );
 }
 
+#[test]
+fn hover_local_function() {
```

Comment:
Could we get tests for different types of imports please?

For example, nested modules, aliased modules, discarded modules that would have the same name, and anything else you can think of.

I think there may be a bug in finding the right import and I would like full test coverage to verify that.

Reply:
You were right! A function marked with the `@external` modifier was not behaving properly

https://github.com/gleam-lang/gleam/blob/4c436129cf3e86bd4d3aede644111582a5a2ee2a/compiler-core/src/language_server/tests/hover.rs#L223

Other than that, I tried to test as many edge cases as possible